content/updates/2020-11/2-DQI-new-missing-image-criteria.md

```diff
@@ -0,0 +1,20 @@
+---
+pim_announcement_alt_img: "TODO"
+pim_announcement_audience:
+- EE
+---
+
+# Data Quality Insights new "Missing image" criteria
+::: meta-data type="New" features="Data quality" available="early November" 
+:::
+
+You all know the expression _"A picture is worth a thousand words"_, don't you?  Images are a critical element in designing compelling product experiences. Data Quality Insights now checks whether your products have an image or not. 
+
+Don’t forget them, otherwise, it will decrease your enrichment grade!
```

Comment:
```suggestion
Don’t forget them, otherwise, it will impact your enrichment grade!
```

Reply:
It feels less clear. I prefer "decrease".